contracts/Pair.sol

```diff
@@ -148,24 +176,46 @@ contract Pair is IPair {
         }
     }
 
-    // Accrue fees on token0
+    // Accrue fees on token0.
     function _update0(uint amount) internal {
-        _safeTransfer(token0, fees, amount); // transfer the fees out to PairFees
-        uint256 _ratio = amount * 1e18 / totalSupply; // 1e18 adjustment is removed during claim
-        if (_ratio > 0) {
-            index0 += _ratio;
+        if (hasGauge == false) {
```

Comment:
@0xMotto can we do if (hasGauge) {} else {} instead of this

Reply:
fixed